src/components/home/CardLimits.tsx

```diff
@@ -0,0 +1,76 @@
+import { marketUSDCAddress, previewerAddress } from "@exactly/common/generated/chain";
+import { borrowLimit, withdrawLimit } from "@exactly/lib";
+import { useQuery } from "@tanstack/react-query";
+import React from "react";
+import { ms } from "react-native-size-matters";
+import { View } from "tamagui";
+import { zeroAddress } from "viem";
+import { useAccount } from "wagmi";
+
+import { useReadPreviewerExactly } from "../../generated/contracts";
+import { getCard } from "../../utils/server";
+import Text from "../shared/Text";
+
+export default function CardLimits() {
+  const { data: card } = useQuery({ queryKey: ["card", "details"], queryFn: getCard });
+  const isCredit = card ? card.mode > 0 : false;
+
+  const { address } = useAccount();
+  const { data: markets } = useReadPreviewerExactly({
+    address: previewerAddress,
+    account: address,
+    args: [address ?? zeroAddress],
+  });
+
+  return (
+    <View display="flex" justifyContent="center" backgroundColor="$backgroundSoft" gap="$s4">
+      <View display="flex" flexDirection="row" justifyContent="space-between" alignItems="center">
+        <Text
+          fontSize={ms(15)}
```

Comment:
instead of setting many props on each `<Text>` component, you should check first if there is an existing variant for that Text. you can check in `components/shared/Text.tsx` if there's a variant that satisfies the figma specifications